osaca/parser/register.py

```diff
@@ -0,0 +1,181 @@
+#!/usr/bin/env python3
+
+from osaca.parser.operand import Operand
+
+
+class RegisterOperand(Operand):
+    def __init__(
+        self,
+        name=None,
+        width_id=None,
```

Comment:
Similar to `InstructionForm`, should be
  - ~`width_id`~ --> `width`
  - ~`prefix_id`~ --> `prefix`
  - ~`regtype_id`~ --> `regtype`

Reply:
https://github.com/RRZE-HPC/OSACA/commit/c2bd4841708deb76492a0054810ab82b7ebea04c